src/BizHawk.Client.EmuHawk/config/HotkeyConfig.Designer.cs

```diff
@@ -198,7 +198,7 @@ private void InitializeComponent()
 			this.AutoScaleDimensions = new System.Drawing.SizeF(6F, 13F);
 			this.AutoScaleMode = System.Windows.Forms.AutoScaleMode.Font;
 			this.CancelButton = this.IDB_CANCEL;
-			this.ClientSize = new System.Drawing.Size(753, 463);
+			this.ClientSize = new System.Drawing.Size(783, 463);

```

Comment:
Are any of these changes necessary? Just changing `iwOffsetX` was sufficient for me with Mono.

Reply:
The window was not wide enough to see all the controls on the General tab when I only changed `iwOffsetX`. The button for "special tricky bindings" was cut off on the third column.